pkg/wrangler/context.go

```diff
@@ -201,7 +240,17 @@ func (w *Context) StartWithTransaction(ctx context.Context, f func(context.Conte
 		return err
 	}
 
-	w.ControllerFactory.SharedCacheFactory().WaitForCacheSync(ctx)
+	timeoutCtx, _ := context.WithTimeout(ctx, cacheSyncTimeout)
```

Comment:
Dropping the cancel func leaks a timer...

https://cs.opensource.google/go/go/+/refs/tags/go1.25.0:src/context/context.go;l=23-25